pkg/cli/manifest/built-in-providers/applications_core.yaml

```diff
@@ -0,0 +1,37 @@
+name: Applications.Core
```

Comment:
Can I suggest `deploy/manifest/built-in-providers` as the home for these files? `pkg` is for go code.